### ISSUE TITLE
Shortcomings

### ISSUE THREAD

**Issue by mrdiega** [partiusfabaa/cs2-MiniAdmin#9]
1) The plugin requires you to indicate the reason for the ban, otherwise there will be a syntax error:
`[ MiniAdmin ] Using: css_ban <userid> <time_seconds> <reason>`
It will be more convenient if it is not necessary to indicate the reason.
2) It is more convenient to indicate the ban time in minutes rather than seconds.
3) The plugin allows you to indicate the reason for the ban in English only. 
For example, like this: `!ban 3 30 читы` will not be banned

**Comment by partiusfabaa** [comment]
1. What's the problem with writing . or 1?
2. -
3. It's true. I don't know what it could be related to.

**Comment by mrdiega** [comment]
> 1. What's the problem with writing . or 1?
> 2. _
> 3. It's true. I don't know what it could be related to.

1. This is not a problem, I wrote it as it would be more convenient for admins. Not everyone is used to indicating the reason for the ban. Typically the syntax looks like this: `css_ban <userid> <time_seconds> [reason]`

While testing the plugin, I found two more bugs. The first is that if you kill all players at once using the !admin command, the server crashes. The second bug is that if a player’s id has a high value, for example 200+ or 500+, then the plugin does not ban him. A message appears in the chat stating that a player with the userid was not found.

**Comment by mrdiega** [comment]
> 1. What's the problem with writing . or 1?
> 2. _
> 3. It's true. I don't know what it could be related to.

1. This is not a problem, I wrote it as it would be more convenient for admins. Not everyone is used to indicating the reason for the ban. Typically the syntax looks like this: `css_ban <userid> <time_minutes> [reason]`

While testing the plugin, I found two more bugs. The first is that if you kill all players at once using the !admin command, the server crashes. The second bug is that if a player’s id has a high value, for example 200+ or 500+, then the plugin does not ban him. A message appears in the chat stating that a player with the userid was not found.
